### PR TITLE
Feature/requirements

### LINKED ISSUE #5: Add requirements info about CPU and memory
Currently, SAPS Scheduler submits simple jobs to the job executor, however information such as cpu and memory can help the executor to better schedule jobs when observing their requirements.

### PR DESCRIPTION
**Pull request by thiagoyeds**
Fixed #5

### CODE REVIEW

File: resources/execution_endtoend_script_tags.json [hunk 1]
```diff
@@ -3,57 +3,81 @@
...
+      "docker_repository":"fogbow/inputdownloader",
+      "memory_usage":"16G",
+      "cpu_usage":"1000m"
     },{
       "name":"nop-download-files",
       "docker_tag":"nop-download-files",
-      "docker_repository":"fogbow/inputdownloader"
+      "docker_repository":"fogbow/inputdownloader",
+      "memory_usage":"16G",
+      "cpu_usage":"1000m"
```
Comment (thiagomanel) [inline resources/execution_endtoend_script_tags.json:20]: what is this `m` in the cpu_usage?

**Review by thiagomanel — CHANGES_REQUESTED** [review]
short question

File: resources/execution_endtoend_script_tags.json (hunk 1 above)
Comment (thiagoyeds) [reply resources/execution_endtoend_script_tags.json:20]: 1000m = 1 vcpu, this is how kubernetes works with the requirements

Comment (thiagomanel) [reply resources/execution_endtoend_script_tags.json:20]: ok, thanks

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**